src/Dijkstra/AST/Branching.agda

```diff
@@ -3,29 +3,20 @@
    Copyright (c) 2022, Oracle and/or its affiliates.
    Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.oracle.com/licenses/upl
 -}
-
```

Comment:
Put the blank line between copyright and `module` back in.
It's easier for users to scan that way.